app/assets/stylesheets/pages/_home.scss

```diff
@@ -1,41 +1 @@
 // Specific CSS for your home-page
-.card-error-message {
- background-color: white;
-padding: 20px;
-}
-
-.add-ride-button {
-margin-top: 20px;
-}
-
-.form-outline {
-  display: flex;
-  align-items: center;
-  background-color: rgba(255, 255, 255, 0.5);
-  padding: 5px;
-  border-radius: 20px;
-  width: 400px;
-}
-
-.form-outline label {
-  margin: 10px;
-  color: #374177;
-
-}
-
-.form-outline input[type="search"] {
-  flex-grow: 1;
-  background-color: transparent;
-  border: none;
-  outline: none;
-  color: white;
-}
-
-.form-outline input[type="submit"] {
-  background-color: #374177;
-  border: none;
-  color: white;
-  padding: 10px 20px;
-  border-radius: 50px;
-  cursor: pointer;
-}
```

Comment:
Hey @mmkclement, did you delete this because it was overheading your css ?  Because I need this part to style the search bar. Sorry, let me know :)

Reply:
Sorry I thought I was freeze at vs code that's why I did some ctrl z. could you repair? if not I will Thanks.